src/index.html

```diff
@@ -23,5 +23,380 @@
 
   <body>
     <h1>Catalog</h1>
```

Comment:
```suggestion

```

Reply:
Not fixed
<img width="850" alt="image" src="https://github.com/user-attachments/assets/2c92fa0b-839e-4a2b-8c60-36d8abb29421">